MBBSEmu/Memory/MemoryCore.cs

```diff
@@ -476,8 +476,13 @@ public void SetByte(ushort segment, ushort offset, byte value)
         /// <param name="segment"></param>
         /// <param name="offset"></param>
         /// <param name="value"></param>
-        public void SetWord(ushort segment, ushort offset, ushort value) =>
-            SetArray(segment, offset, BitConverter.GetBytes(value));
+        public void SetWord(ushort segment, ushort offset, ushort value)
+        {
+            _ = new Span<byte>(_memorySegments[segment])
```

Comment:
does this allocate a new object or is this something like a stack variable?

Reply:
Span<T> lives on the stack and gives us a ref to the dictionary value. This avoids multiple calls to the dictionary indexer which performs a hash lookup.